content/docs/command-reference/commit.md

```diff
@@ -21,9 +20,9 @@ positional arguments:
 The `dvc commit` command is useful for several scenarios, when data already
 tracked by DVC changes: when a [stage](/doc/command-reference/run) or
 [pipeline](/doc/command-reference/dag) is in development/experimentation; when
-manually editing or generating DVC <abbr>outputs</abbr>; or to force DVC-file
-updates without reproducing stages or pipelines. These scenarios are further
-detailed below.
+manually editing or generating DVC <abbr>outputs</abbr>; or to force update
+DVC-tracked files without reproducing stages or pipelines. These scenarios are
+further detailed below.
```

Comment:
"force update DVC-tracked files" is not right either 😢 `commit` doesn't change neither stages nor the files/dirs tracked by DVC. Message me on Discord if you're unsure for the next round please, so we can go over this bump. Thanks

Reply:
Thanks for the help. I have updated the doc and fixed some grammar issues.

Reply:
Great, thanks. Still seeing some grammar issues... Will make another review.